website/allowed-subcategories.txt

```diff
@@ -1,4 +1,5 @@
 Activity Tracker API
+Admin Service API
```

Comment:
Not required. Let's remove this. We already have `Security and Compliance Center` in this file.